src/RegExLib.Core/Entities/Expression.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using RegExLib.Core.Events;
+using RegExLib.Core.Interfaces;
+using RegExLib.SharedKernel;
+
+namespace RegExLib.Core.Entities
+{
+    public class Expression : BaseEntity
+    {
+        public int AuthorId { get; private set; }
+        public string Title { get; }
+        public string Pattern { get; }
+        public string Description { get; }
+        public Author Author { get; private set; }
+
+        public Expression(string title, string pattern, string description)
+        {
+            Title = title;
+            Pattern = pattern;
+            Description = description;
+        }
+
+        public void SetAuthor(Author author)
```

Comment:
I'm not sure this is needed - I don't think there's a use case for changing authors, generally. But if there is, I'm not sure it belongs here. It seems like that would span aggregates and thus might need to be a domain service. But YAGNI means let's just leave it out for now.